api/rest/openapi.yaml

```
@@ -1,359 +1,204 @@
-      enum: [ID, UPDATED_AT, TYPE, PROTOCOL]
-tags:
-  - name: Devices
-    description: Device management endpoints
-  - name: Scenarios
-    description: Scenario orchestration endpoints
-security:
-  - bearerAuth: []
+        name:
+          type: string
+        description:
+          type: string
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)